Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -529,6 +620,207 @@ def submit_false_positive_report_command(client, args):
     return command_results
 
 
+def get_a_list_of_vendors_command(client, args):
+    page_size = str(args.get('page_size', ''))
+    page_number = str(args.get('page_number', ''))
```

Comment:
For int-based parameters, we have a `arg_to_number` CommonServerPython function that prints proper errors for users if an invalid input is used (anything that isn't a number). It returns an int, though.

I'd recommend passing int parameters as an int type between functions, not a string. Then convert them when sending HTTP requests (I think it might be done automatically by `requests` though, so you might not even need to do that).

The same note applies for other cases in other functions below.